### DISCUSSION TITLE
delete button on GUI doesnt delete ticked files

### DISCUSSION THREAD

**Discussion by blackfrank74** [qarmin/czkawka#1104] [General]
Ubuntu 22.04, app v6.0.0

I run a search for similar videos.
List of possible similarities appear.
I review and tick a file I want to delete.

![image](https://github.com/qarmin/czkawka/assets/34759786/fe413e13-2ac8-4114-b1a9-d97f43e822fc)

Step 1:
I click the delete button on the bottom righ(ish) part of the GUI.
Nothing happens.
I run a file search in the OS, I can see the 2 files being found

Troubleshooting I've tried -
1. I've exited and restarted the app - did not fix the issue
2. I've removed  and reinstalled  - did not fix the issue
3. Ive rebooted  - did not fix the issue

Any ideas?

**Comment by rugk** [comment]
Did you see a confirmation dialog asking for deletion?

**Comment by blackfrank74** [comment]
Hi - I did not. 

However, I got this response to the related issue I raised 
https://github.com/qarmin/czkawka/issues/1105#issuecomment-1774465165

Based on that response I 
- went to https://github.com/qarmin/czkawka/releases/
- downloaded linux_czkawka_gui from the assets section into my downloads folder
- set it to executable
- launched it - double click
- ran the "similar videos" search
- clicked delete which brought up a confirm box, and then deleted the files.

So all seems good now.
